releases/Process/Major Release Process.md

```diff
@@ -59,91 +56,108 @@ The release process is managed by sig-release. To request an update to this docu
 
-Exit Criteria: All Pre-Release Steps are complete
+*Exit Criteria: All Pre-Release Steps are complete*
 
 ## Phase (3 of 4): Release Day
-*Intent: Accomplish all tasks needed to deploy the release so that it is available for download.* 
-Begins day of release
-Entry Criteria: All pre-release steps are complete.
+*Intent: Accomplish all tasks needed to deploy the release so that it is available for download. 
+Begins: day of release
+Entry Criteria: All pre-release steps are complete. Prior to Release Day you will have had 2 opportunities to go through the Release day work (1) In the Planning phase of the release, there will be a task "Prepare for Launch Call" which will be the first time you read through all of the steps in the launch call (2) In the Pre-Release phase there will be a "dry run" of the release day call.*
 
 ### 3.1 Steps
-//TODO - document the process for Release day and running the call
+1. Hold the release call on Release Day. You should have an issue in your project in the "Release Day' phase, called "Execute Launch for O3DE...", similar to [this task for the 22.10.0 Release](https://github.com/o3de/o3de/issues/11096).
+2. After roll call is taken, ask if anyone on the call has a reason to postpone the release. Discuss any concerns. 
+3. During the call, walk through the steps in your Release task. Ask each owner to perform their task and ask them to confirm once they are done. The work for LFS Binary Objects is expected to take a while, and will not be complete during the call. This is OK.
+4. The final steps involve verification. After verification, the release is done. At this point confirm with everyone that they do not have any remaining work. Confirm the plan for LFS Binary Objects with the owner - basically you need them to confirm with you once that work is complete. Congratulations, the release is now available to the community!
+5. In Discord #marketing-committee post a note informing them that the release is available. 
 
-Exit Criteria: All Release Day Tasks are complete.
+*Exit Criteria: All Release Day Tasks are complete.*
 
 ## Phase (4 of 4): Post-Release
 *Intent: Perform activities that typically occur after release day.* 
-Begins immediately after the release. 
-
-//TODO outline the things that take place post-release
-
-Creating the stabilization branch
+Begins immediately after the release. The work in this phase may be minimal, depending on the release,
 
+### 4.1 Steps
+1. Go through the "Post Release" phase of the Github project board and ensure the tasks get completed. Reach out to the owners of each task as needed to make sure they complete their tasks.
+2. Make sure all previous tasks are fully completed closed (you may have minor things you wanted to follow up on).
+3. Make any updates to this documentation as needed. This documentation should reflect the current release process.
```

Comment:
@tonybalandiuk Should we add a step to gather feedback for the release, just like we did after the 22.10 release?

Reply:
excellent point. Will add.

Reply:
included it in my Jan 3 commit